### PR TITLE
Edit schedule

### PR DESCRIPTION
**Pull request by lizekai-richard**

### CODE REVIEW

File: src/main/java/seedu/address/model/module/schedule/Schedule.java [hunk 1]
```diff
@@ -89,14 +113,20 @@ public static boolean isTimeValid(String startTime, String endTime) {
...
             return false;
         }
-        return true;
+        return newSchedule.weekday.equals(this.weekday);
+    }
+
+    /**
+     * Checks if the module code and class type of another schedule match the module code
+     * and class type of current schedule.
+     * @param other another schedule
```
Comment (drkkjh) [inline src/main/java/seedu/address/model/module/schedule/Schedule.java:126]: Should this be `other schedule` or `another schedule`?

**Review by drkkjh — COMMENTED** [review]
LGTM except for a small nitpicky issue with a javadoc comment! :smile:

**Review by myangat0343 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/module/schedule/Schedule.java (hunk 1 above)
Comment (lizekai-richard) [reply src/main/java/seedu/address/model/module/schedule/Schedule.java:126]: Yes. Thanks for pointing out!

**Pull request merged**